cypress/integration/instruments.ts

```diff
@@ -161,15 +163,34 @@ context('Instrument tests', () => {
 
     cy.createCall(call2);
 
+    cy.navigateToTemplatesSubmenu('Proposal templates');
+
+    cy.contains('default template')
+      .parent()
+      .get("[title='Edit']")
+      .first()
```

Comment:
I think you don't need `.first()` here at all. There is only one edit button when you get it by parent.